### DISCUSSION TITLE
📛Badges Coming in January!!📛

### DISCUSSION THREAD

**Discussion by codemillmatt** [microsoft/hack-together-dotnet#118] [Announcements]
Hello all!

Thank you to everyone who submitted a project for the Great .NET 8 Hack! Here's the [blog post](https://devblogs.microsoft.com/dotnet/great-dotnet-8-hack-winners/) where we announced the winners of the hack, and I wanted to let all the participants know that you will be getting a digital badge yet in January! So please hang tight just a bit longer while we finish up the work on it.

Thank you again!

**Comment by boku-engr0** [comment]
It think it's getting too late now. Its February already

**Comment by YuriyMorozyuk95** [comment]
@codemillmatt Hi does badges was already send?
 can not find anything in my outlook(

**Comment by shreenivaska** [comment]
Badge??????? its march already

**Reply by ghost** [reply]
and you're not alone. I still haven't received the badge.

**Comment by AdityaSkhorne** [comment]
Haven't received yet

**Comment by gauravpathak91** [comment]
Hey @codemillmatt I still haven't received my badge. My teammates already got theirs.

**Comment by Angela95** [comment]
Same here @codemillmatt, still waiting for badge. My teammate got it a few days ago.
